packages/toolpad-app/src/server/functionsTypesWorker.ts

```diff
@@ -413,6 +413,7 @@ export default async function extractTypes({
   const checker = program.getTypeChecker();
 
   const usingCreateFunction = [];
+  let createFunctionWarningShown = false;
```

Comment:
Shouldn't this go in module scope? Otherwise it will log every time we call this function

Reply:
Also, have you tried just wrapping `createFunction` in `toolpad-core` with `util.deprecate`?

Reply:
I get https://vitejs.dev/guide/troubleshooting.html#module-externalized-for-browser-compatibility when trying to use `util` to wrap `createFunction`. 

Another benefit is being able to pinpoint how many functions are using the deprecated function, which is a helpful addition to the warning in my view